Content/Items/Gear/Gear.cs

```diff
@@ -500,5 +526,27 @@ public static string GetDescriptor(GearType type, GearRarity rare, GearInfluence
 
 			return $" {influenceName}{rareName}{typeName}";
 		}
+		
+		public override void ModifyHitNPC(Player player, NPC target, ref NPC.HitModifiers modifiers)

```

Comment:
having modifiers take effect here is bound to become monolithic. I think we should adopt something similar to starlight's architecture where global hooks are exposed as events and subscribed to by modifiers, or handle some sort of unified statistics somewhere for the player.